chore: update meraki options and formatting (#14437)
newrelic/docs-website: src/content/docs/network-performance-monitoring/advanced/advanced-config.mdx

      <tbody>
        <tr>
          <td>
            eapi_config.username
          </td>
          <td>
            ✓
          </td>
          <td>
            The username to pass to the device to authenticate the eAPI authentication.
          </td>
        </tr>
        <tr>
          <td>
            eapi_config.password
          </td>
          <td>
            ✓
          </td>
          <td>
            The password to pass to the device to authenticate the eAPI authentication.
          </td>
        </tr>
        <tr>
          <td>
            eapi_config.transport
          </td>
          <td/>
          <td>
            Specifies the type of connection transport to use. The possible values are `https` and `http`. Default: `https`.
          </td>
        </tr>
        <tr>
          <td>
            eapi_config.port
          </td>
          <td>
            ✓
          </td>
          <td>
            The TCP port of the endpoint for the eAPI connection.
          </td>
        </tr>
      </tbody>
    </table>
  </Collapser>

  <Collapser
    id="meraki"
    title="Meraki Dashboard API"
  >

>     The [Meraki Dashboard API](https://developer.cisco.com/meraki/api-latest/) integration pulls various metrics related to the health of your Meraki environment. The combination of various configuration options allow you to setup different monitoring scenarios for your needs.

Reviewer: ```suggestion
    The [Meraki Dashboard API](https://developer.cisco.com/meraki/api-latest/) integration pulls various metrics related to the health of your Meraki environment. The combination of various configuration options allows you to set up different monitoring scenarios for your needs.
```

          <td> Monitor client status and performance per network. *(Not recommended for large environments due to timeout problems)* </td>
        </tr>
        <tr>
          <td> meraki_config.monitor_devices </td>
          <td />
          <td> true | false (Default: false) </td>
          <td> Monitor the status of every Meraki device in the organization. </td>
        </tr>
        <tr>
          <td> meraki_config.monitor_org_changes </td>
          <td />
          <td> true | false (Default: false) </td>
          <td> Monitors the change log for the organization. </td>
        </tr>
        <tr>
          <td> meraki_config.monitor_uplinks </td>
          <td />
          <td> true | false (Default: true) </td>
          <td> Monitors the uplink status of every Meraki MX, MG and Z series device in the organization. </td>
        </tr>
        <tr>
          <td> meraki_config.monitor_vpn_status </td>
          <td />
          <td> true | false (Default: false) </td>
          <td> Monitors the VPN statuses across the networks in the organization. </td>
        </tr>
      </tbody>
    </table> 
    
    ### Filtering options
    

>     These options allow you to restrict monitoring to specifically-targeted objects in your Meraki environment.

Reviewer: ```suggestion
    These options allow you to restrict monitoring to specifically targeted objects in your Meraki environment.
```

          <th> Required </th>
          <th> Input </th>
          <th> Description </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td> meraki_config.organizations </td>
          <td />
          <td> Regex in [RE2 syntax](https://github.com/google/re2/wiki/Syntax) (Default: null) </td>
          <td> Filters all monitoring to a specific list of organizations. </td>
        </tr>
        <tr>
          <td> meraki_config.networks </td>
          <td />
          <td> Regex in [RE2 syntax](https://github.com/google/re2/wiki/Syntax) (Default: null) </td>
          <td> Filters all monitoring to a specific list of networks. </td>
        </tr>
        <tr>
          <td> meraki_config.product_types </td>
          <td />
          <td> Valid types are wireless, appliance, switch, systemsManager, camera, cellularGateway, sensor, and cloudGateway. (Default: null) </td>
          <td> Adds parameters to the [monitor_devices](https://developer.cisco.com/meraki/api-latest/get-organization-devices-statuses/) API request to filter on specific types of devices. </td>
        </tr>
      </tbody>
    </table>
    
    ### Additional preferences
    

>     These options allow you to further-define the data collected from the main configuration options. Various combinations are described in the examples section above. 

Reviewer: ```suggestion
    These options allow you to further define the data collected from the main configuration options. Various combinations are described in the examples section above. 
```
